pkg/virt-controller/watch/vmi.go

```diff
@@ -269,6 +269,30 @@ func (c *VMIController) execute(key string) error {
 
 }
 
+// verifies all conditions match even if they are not in the same order
+func conditionsEqual(a []virtv1.VirtualMachineInstanceCondition, b []virtv1.VirtualMachineInstanceCondition) bool {
```

Comment:
Perhaps, []virtv1.VirtualMachineInstanceCondition needs to be a map in the feature?

Reply:
That would break the client support in kubectl and all other generic k8s code which supports conditions.